client/src/Components/DropDown.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+
+function DropDown() {
+  const [product, setProduct] = useState();
+  return (
+    <div>
```

Comment:
The select element shall come with a name attribute and a label as well ;)
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select